README.md

```diff
@@ -1,157 +1,44 @@
-CKEditor&nbsp;5 [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Check%20out%20CKEditor%205%20on%20GitHub&url=https%3A%2F%2Fgithub.com%2Fckeditor%2Fckeditor5)
-===================================
+# CKEditor&nbsp;5 Fork
 
-[![npm version](https://badge.fury.io/js/ckeditor5.svg)](https://www.npmjs.com/package/ckeditor5)
-[![Coverage Status](https://coveralls.io/repos/github/ckeditor/ckeditor5/badge.svg?branch=master)](https://coveralls.io/github/ckeditor/ckeditor5?branch=master)
-[![Build Status](https://travis-ci.com/ckeditor/ckeditor5.svg?branch=master)](https://app.travis-ci.com/github/ckeditor/ckeditor5)
-![TypeScript Support](https://badgen.net/badge/Built%20With/TypeScript/blue)
+This is a forked version of CKEditor 5 with additional customizations and features. The original repository can be found at [ckeditor/ckeditor5](https://github.com/ckeditor/ckeditor5).
```

Comment:
Remove this part:

"The original repository can be found at [ckeditor/ckeditor5](https://github.com/ckeditor/ckeditor5)."